recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+    folder: cugraph
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}a.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.12.4
+    - make
+  host:
+    - boost-cpp >=1.66
+    - nccl >=2.8.4
+    - ucx-proc =*=gpu
+    - libfaiss =*=*_cuda
+    - spdlog =1.7.0
+  run:
+    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - nccl >=2.8.4
+    - ucx-proc =*=gpu
+    - libfaiss =*=*_cuda
```

Comment:
`libfaiss` uses [`run_exports`]( https://github.com/conda-forge/faiss-split-feedstock/blob/66dd1fd60fc10326453c7d020c16e58b6200825a/recipe/meta.yaml#L146 ). So we can drop this as it will be added automatically

```suggestion
```

Reply:
dropped in next commit.